stablehlo/dialect/StablehloOps.td

```diff
@@ -433,7 +433,8 @@ def StableHLO_Log1pOp: StableHLO_UnaryElementwiseOp<"log_plus_one",
 }
 
 def StableHLO_LogisticOp: StableHLO_UnaryElementwiseOp<"logistic",
-    [Pure, HLO_CompatibleOperandsAndResultType], HLO_FpOrComplexTensor> {
+    [Pure, HLO_CompatibleOperandsAndResultType /*logistic_c1*/],
+    HLO_FpOrComplexTensor /*logistic_i1*/> { /*logistic_c1*/
```

Comment:
The updated guideline has to account for writing comments related to shape inference. Since this is hidden by the `StableHLO_UnaryElementwiseOp` class, we have another comment on line 437 as having them inside StableHLO_UnaryElementwiseOp would make it messy. The ops you mentioned (`LogOp` and `AbsOp`) were checked in before the updated guideline, so there is a discrepancy which would be addressed in #1002.